### DISCUSSION TITLE
Playlists not syncing(?)

### DISCUSSION THREAD

**Discussion by ghost** [libre-tube/LibreTube#2734] [General]
Hey, ive an account on an instance of piped, yet the playlists i create and add videos to on libretube dont seem to sync to the actual servers playlists. i suspect this is because a local playlist is used instead of the instance one. a toggle or property to set for playlists to sync with the instance's playlists would be very good for this.

of course, it could just be a weird misconfiguration on my part but im relatively certain im logged in correctly and so on

thanks!

**Comment by Bnyro** [comment]
Navigating to settings -> instance contains an option called "log out"?
If not, you're not logged in with an account, I'd recommend making a backup of your playlists then, signing into an account and then importing the backup you made previously.

**Comment by ghost** [comment]
there is a logout option, im definitely logged in yet still it seems the libretube playlist differs

**Comment by Bnyro** [comment]
As long as you're logged in at both, Piped in the browser and LibreTube on your phone with the same account on the same Piped instance, there won't be any difference between the playlists. So either you're using different instances or different accounts.

**Comment by Bnyro** [comment]
When logged in with an account, playlists won't ever be stored locally, so having a playlist on LibreTube that doesn't exist on Piped is impossible and vice versa.

**Comment by ghost** [comment]
do i have to be logged into piped on my phone browser? that would maybe explain it, since im only logged into the website on my pc and not on phone 

edit: i logged in on browser on phone and it does seem to match up now, will check the pc soon

**Comment by ARBoyGo** [comment]
You need to know, Some instances doesn't support Playlist creation...

**Comment by Bnyro** [comment]
> You need to know, Some instances doesn't support Playlist creation...
>
That's not true, all instances where you're able to register support it.

**Comment by Bnyro** [comment]
> do i have to be logged into piped on my phone browser? 
>
No, only in LibreTube itself.
